homework_11/homework_11.py

```diff
@@ -0,0 +1,78 @@
+# # 1 Написати власний декоратор, задачею якого має бути друк назви функції і
+# # часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+# import time
+
+
+# def deco_time_and_name(func_to_deco):
+#     def print_time_and_name():
+#         str_name = func_to_deco.__name__
+#         print(f"function name - {str_name}")
+#         str_time = time.ctime()
+#         print(f"start function time - {str_time}")
+#         func_to_deco()
+#
+#     return print_time_and_name
+#
+#
+# @ deco_time_and_name
+# def standard_func():
+#     print("this is a standard function")
+#
+# standard_func()
+
+
+
+# 2 Написати кастомний Exception клас, MyCustomException, який
+# має повідомляти "Custom exception is occured".
+
+# class MyCustomException(Exception):
+#     print("Custom exception is occured")
+#
+#
+# raise MyCustomException()
+
+
+
+# # # 3 Написати власний менеджер контексту, задачею якого буде
+# # # друкувати "==========" – 10 знаків дорівнює перед виконанням коду та
+# # # після виконання коду, таким чином виділяючи блок коду символами дорівнює.
+# # У випадку виникнення будь-якої помилки вона має бути надрукована текстом,
+# # проте програма не має завершити своєї роботи.
+# #
+
+# class MyManager:
+#     def __enter__(self):
+#         print("==========")
+#         try:
+#             pass
+#         except Exception as oll_exception:
+#             print((f'this exception: {oll_exception}'))
+#
+#     def __exit__(self, exc_type, exc_val, exc_tb):
```

Comment:
You need to check if there is some exception (`if exc_type:`) and print its value (`exc_val`). And add `return True` at the end to avoid exception raising